src/shell-integration/bash/ghostty.bash

```diff
@@ -95,6 +95,78 @@ if [[ "$GHOSTTY_SHELL_FEATURES" == *"sudo"* && -n "$TERMINFO" ]]; then
   }
 fi
 
+# SSH Integration
+if [[ "$GHOSTTY_SHELL_FEATURES" =~ ssh-(env|terminfo) ]]; then
+
+  if [[ "$GHOSTTY_SHELL_FEATURES" =~ ssh-terminfo ]]; then
+    readonly _CACHE="${GHOSTTY_RESOURCES_DIR}/shell-integration/shared/ghostty-ssh-cache"
+  fi
+
+  # SSH wrapper
+  ssh() {
+    local env=() opts=() ctrl=()
```

Comment:
It's not a big thing either way, but it might be helpful to prefix these particular local variables with `ssh_` to communicate that these are the values we're going to use when executing `ssh` itself.

```suggestion
    local ssh_env=() ssh_opts=() ssh_ctrl=()
```